### PR TITLE
Add JSON view for list_pages

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by pascalchevrel** [comment]
Could you add a functional test? I like to make sure all json sources are always working. Thanks

File: app/views/listpages_api.inc.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+$export_data = [];
+foreach ($displayed_sites as $site_index => $current_website) {
+    $website_name = Project::getWebsiteName($current_website);
+    $website_data_source = Project::getWebsiteDataType($current_website);
+
+    foreach (Project::getWebsiteFiles($current_website) as $current_filename) {
+        $displayed_filename = ($website_data_source == 'lang') ?
+                              $current_filename :
+                              basename($current_filename);
```
Comment (pascalchevrel) [inline app/views/listpages_api.inc.php]: nit: I prefer that we stick to this coding style convention:
```php
       $displayed_filename = ($website_data_source == 'lang')
            ? $current_filename
            : basename($current_filename);
```

**Comment by pascalchevrel** [comment]
LGTM

**Comment by flodolo** [comment]
Sure, let me look into the functional test.

**Comment by flodolo** [comment]
@pascalchevrel Want to take a quick look at the new commit 7ccdf3a?

**Comment by pascalchevrel** [comment]
r+

**Pull request merged**